pinot-controller/src/main/resources/app/pages/TenantDetails.tsx

```diff
@@ -466,6 +466,20 @@ const TenantPageDetails = ({ match }: RouteComponentProps<Props>) => {
     const result = await PinotMethodUtils.rebalanceBrokersForTableOp(tableName);
     syncResponse(result);
   };
+  
+  const handleRepairTable = () => {
+    setDialogDetails({
+      title: 'Repair Table',
+      content: 'Are you sure want to repair the table?',
```

Comment:
please add more details in the content on what this action does.

Reply:
done